docs/source/index.rst

```diff
@@ -58,6 +58,31 @@ the usage notes.
 
 A list of current issues can be found `here <https://github.com/dask/fastparquet/>`_.
 
+Relation to Other Projects
+--------------------------
+
+-  `parquet-cpp <https://github.com/apache/parquet-cpp/>`_ is a low-level C++
+implementation of the parquet format which can be called from python using
+Apache `Arrow <http://pyarrow.readthedocs.io/en/latest/>`_ bindings. We aim to
+collaborate with parquet-cpp, and it is possible, in the medium term, that their low-level
+routines will replace some functions in fastparquet or that high-level logic in
+fastparquet will be migrated to C++.
```

Comment:
I'm not sure we should claim that we aim to collaborate.  It's not clear that this will or will not happen.  Perhaps instead "it's possible that in the future ..."